doc/architecture-design.md

```diff
@@ -188,7 +188,7 @@ Note: When both Route and Service enable the same plugin, the Route parameter ha
 
 The `Plugin` configuration can be bound directly to `Route` or it can be bound to `Service` or `Consumer`. For the configuration of the same plugin, only one copy is valid, and the configuration selection priority is always `Consumer` > `Route` > `Service`.
 
-In `conf/config.yaml`, you can declare which plugins are supported by the local APISIX node. This is a whitelisting mechanism. Plugins that are not in this whitelist will be automatically ignored. This feature can be used to temporarily turn off or turn on specific plugins, which is very effective in dealing with unexpected situations.
+In `conf/config.yaml`, you can declare which plugins are supported by the local APISIX node. This is a whitelisting mechanism. Plugins that are not in this whitelist will be automatically ignored. This feature can be used to temporarily turn off or turn on specific plugins, which is very effective in dealing with unexpected situations. If you want to add new plugins based on existing plugins, you need to copy the `plugins` node content of `config-default.yaml` to the `plugins` node of `conf/config.yaml`.
```

Comment:
I think `config-default.yaml` is changed to `conf/config-default.yaml`. This is better, keep it consistent with `conf/config.yaml`, and also indicate in which directory the `config-default.yaml` file is located. What do you think?

Reply:
I agree, fixed.

Reply:
@idbeta I do not think you have fixed, please check this